### PR TITLE
Update onboard-windows-server.md

### PR DESCRIPTION
**Pull request by msbemba**
Servers can be onboarded through SCCM (2207 now even supports onboarding with Unified Agent), local script, GP/Group Policy script, Defender for Cloud integration etc.
updated/removed the note

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9466

File: microsoft-365/security/defender-endpoint/onboard-windows-server.md [hunk 1]
```diff
@@ -47,8 +47,6 @@ You'll need to complete the following general steps to successfully onboard serv
 
 :::image type="content" source="images/server-onboarding.png" alt-text="Server Onboarding" lightbox="images/server-onboarding.png":::
 
-> [!NOTE]
-> Servers are on-boarded using GPs only.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/onboard-windows-server.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**